meta/contributor-handbook.md

```diff
@@ -0,0 +1,88 @@
+
+### In a Nutshell
+
+| \# | Name | Description | Time to merge |
+| ---- | ---- | ---- | ---- |
+| 1 | Initial | Contains an initial pattern idea or what we call a donut (a pattern with missing sections). | A few days / weeks |
+| 2 | Structured | Contains a complete pattern that it is not properly validated yet (e.g. because it only synthesizes an idea or the experiences from one instead of three organizations). | A few days / weeks |
+| 3 | Validated | The pattern is validated (e.g. because three or more instances exists and are synthesized by the pattern) and its writing is of high quality. | One or two months |
+| 4 | Established | The pattern is validated, its writing is of "text book" quality, and the community agrees it is responsible to use it as teaching material. | As long as it takes |
+
+For the first pattern you contribute, you should aim for maturity levels `1: initial` or `2: structured`. 
+
+If you want to help promoting a pattern one maturity level up, we suggest to first create an issue to discuss the matter and see if someone else is working on that already. After that, you can create a pull request with the necessary changes.
+
+To achieve a given maturity level a pattern has to satisfy the requirements for that given maturity level and lower levels. The following sections lay out the requirements per maturity level in detail.
+
+
+### Requirements: Level 1 - Initial
+
+Patterns (or other documents) of level 1 are stored in the directory `/1-initial`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	- Donuts - patterns without solution or other missing sections (`donut-*.md`)
+	- Tale - unstructured information in any form (`tale-*.md`)
+	
+- Validation requirements:
+	- N/A
+	
+- Content requirements:
+	- The document is readible & comprehensible for other's than the author(s)
+	- The author(s) contribute the contents according to the [license](../LICENSE.txt) & are allowed to do so
+	- Thoughts and contents by third parties are quoted / referenced explicitly
+
+	
+### Requirements: Level 2 - Structured
+
+Patterns of level 2 are stored in the directory `/2-structured`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	
+- Validation requirements:
+	- Is validated by at least one known instance 
+		- Alternatively: key elements of the pattern have been validated in separate contexts and, in consequence, it is justified to believe the full solution will function
+	
+- Content requirements:
+	- Complies with the [patterns format](pattern-template.md)
+	- Complies with *basic style guide*(#) - *Oops! We have not yet developed this*
+	
+	
+### Requirements: Level 3 - Agreed
+
+Patterns of level 3 are stored in the directory `/3-agreed`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	
+- Validation requirements:
+	- Is validated by at least three known instances
+	- Considers all known instances to the best of working group members' knowledge

```

Comment:
How this this point different from "Is validated by at least three known instances"?

Reply:
If it's validated by three instances, there can still be `x` instances that are doing something that is conflicting with the presented pattern. So, for this maturity level, it should not have such conflicts. Too strict?

Reply:
This is fine by me, let's keep it.
Feel free to resolve this thread.